plugins/smartcard/msd-smartcard-plugin.c

```diff
@@ -36,16 +34,15 @@
 
 struct MsdSmartcardPluginPrivate {
         MsdSmartcardManager *manager;
-        DBusGConnection     *bus_connection;
-
+        GDBusProxy          *screensaver_proxy;
         guint32              is_active : 1;
 };
 
 typedef enum
 {
-    MSD_SMARTCARD_REMOVE_ACTION_NONE,
-    MSD_SMARTCARD_REMOVE_ACTION_LOCK_SCREEN,
-    MSD_SMARTCARD_REMOVE_ACTION_FORCE_LOGOUT,
+        MSD_SMARTCARD_REMOVE_ACTION_NONE,
+        MSD_SMARTCARD_REMOVE_ACTION_LOCK_SCREEN,
+        MSD_SMARTCARD_REMOVE_ACTION_FORCE_LOGOUT,
```

Comment:
If you do not make any changes to the code, please do not format it, as you will increase the number of lines to review.